lxd/auth/oidc/oidc.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/canonical/lxd/shared"
 )
 
+const (
+	CookieNameIDToken      = "oidc_identity"
```

Comment:
These are exported symbols so I'd have expected a linter to complain about missing comments here.

Reply:
These constants don't actually need to be exported. In any case, after some investigation today I've found that the `revive` linter isn't running in golangci-lint. I think we should turn it on. In addition, when the `revive` linter is run via golangci-lint, it doesn't seem to catch missing comments on exported methods, functions, variables, or constants and we need to run it separately to specifically find those cases. I'm working through it now but there is an awful lot to fix.

Reply:
I've unexported these but added comments for them in any case. I'm working on a separate PR to fix the linter config.